integration_tests/tests/rekognition.rs

```diff
@@ -0,0 +1,17 @@
+#![cfg(feature = "rekognition")]
+
+extern crate rusoto_core;
+extern crate rusoto_rekognition;
+
+use rusoto_rekognition::{Rekognition, RekognitionClient, ListCollectionsRequest};
+use rusoto_core::{DefaultCredentialsProvider, Region, default_tls_client};
+
+#[test]
+fn should_list_collections() {
+    let credentials = DefaultCredentialsProvider::new().unwrap();
+    let client = RekognitionClient::new(default_tls_client().unwrap(), credentials, Region::UsEast1);
+    let request = ListCollectionsRequest::default();
+
+    let result = client.list_collections(&request).unwrap();
+	println!("{:#?}", result);
```

Comment:
Hmm, I just copied this from [the original PR](https://github.com/rusoto/rusoto/blob/1336fdd196d292c6e767e6730e1974e7fe9435e8/tests/rekognition.rs), but I can fix it.